index.js

```diff
@@ -18,6 +18,10 @@ var fs = require('fs');
 var path = require('path');
 var builtins = require('browserify/lib/builtins');
 
+builtins.dns = require.resolve('runtime-node-dns');
+builtins.net = require.resolve('runtime-node-net');
+builtins.http = require.resolve('http-node');
```

Comment:
ah, you're right, browserify does the same https://github.com/substack/node-browserify/blob/master/lib/builtins.js